src/lib/zapUtils.ts

```
@@ -0,0 +1,78 @@
+    } else if (lud16) {
+      // Extract domain and name from the LUD16 address
+      let [name, domain] = lud16.split('@');
+      // Build the LNURL service URL dynamically based on the domain
+      lnurl = new URL(`/.well-known/lnurlp/${name}`, `https://${domain}`).toString();
+    } else {
+      return null; // Handle missing LNURL case
+    }
+
+    console.log("LNURL:", lnurl);
+
+    let res = await _fetch(lnurl);
```

Biome

[error] This let declares a variable that is only assigned once.

'res' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)